presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -428,6 +428,38 @@ public void testUnsupportedColumnType()
         session.execute("DROP KEYSPACE keyspace_6");
     }
 
+    public void testNestedCollectionType()
+    {
+        session.execute("CREATE KEYSPACE keyspace_7 WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor': 1}");
```

Comment:
Can we keyspace_7 -> keyspace_test_nested_collection (or other meaningful name)?

Reply:
Sure, let me update it.